platform/mellanox/sdk.mk

```diff
@@ -1,4 +1,4 @@
-MLNX_SDK_BASE_URL = https://github.com/Mellanox/SAI-Implementation/raw/310eb922e133ecb3243e10efa4440fabe0ca73c7/sdk
+MLNX_SDK_BASE_URL = https://github.com/Mellanox/SAI-Implementation/raw/d5810ae692f47898bbf6e093875c89b5f7e4ef19/sdk
```

Comment:
can we use MLNX_SAI_VERSION which is export from mlnx-sai.mk instead of using hash values again?